### PR TITLE
Fix #2399 - Remove redundant drawable ic_search_black_24dp

### LINKED ISSUE #2399: Remove redundant drawable ic_search_black_24dp
**Actual Behaviour**

Drawable file ic_search_black_24dp.xml is currently not being used and hence is redundant.

**Expected Behaviour**

Drawable file ic_search_black_24dp.xml should not be present.

**Steps to reproduce it**

Delete ic_search_black_24dp.xml file

**Would you like to work on the issue?**

Yes, I would like to work on the issue.

### PR DESCRIPTION
**Pull request by Princejhabak**
Fixed #2399 

Changes: Unused drawable file ic_search_black_24dp.xml is deleted.

### CODE REVIEW

File: app/src/main/res/values/styles.xml [hunk 1]
```diff
@@ -215,7 +215,7 @@
         <!-- Close button icon -->
         <item name="closeIcon">@drawable/ic_close_black_24dp</item>
         <!-- Search button icon -->
-        <item name="searchIcon">@drawable/ic_search_black_24dp</item>
+        <item name="searchIcon">@drawable/ic_search_white_24dp</item>
```
Comment (abishekvashok) [inline app/src/main/res/values/styles.xml]: Err why did you remove it

Comment (Princejhabak) [reply app/src/main/res/values/styles.xml]: @abishekvashok, sorry but you are addressing the drawable file or styles.xml

Comment (abishekvashok) [reply app/src/main/res/values/styles.xml]: Styles.xml

Comment (Princejhabak) [reply app/src/main/res/values/styles.xml]: @abishekvashok, if the drawable file ic_search_black_24dp is removed then referencing it in styles.xml will produce error.  And drawable file ic_search_black_24dp must be deleted as it is currently not used anywhere in the project.

Comment (abishekvashok) [reply app/src/main/res/values/styles.xml]: The drawable was being used. You removed it's use and deleted it. See the entire diff

Comment (Princejhabak) [reply app/src/main/res/values/styles.xml]: @abishekvashok, I had to remove the use of the drawable file because of this

screenshot_20190119-102052


So I had to create ic_search_white_24dp
